source/includes/administration/_service_connections.md

```diff
@@ -59,3 +59,92 @@ Attributes | &nbsp;
 `name`<br/>*string* | The name of the service connection
 `type`<br/>*string* | The type of the service connection.
 `status`<br/>*Object* | Status of the service connection. Tells you if the service is up.<br/>*includes*: `lastUpdated`, `reachable`
+
+<!-------------------- GET APIINFO -------------------->
+
+### Retrieve API credentials
+
+`GET /{serviceConnectionId}/apiInfo`
```

Comment:
The API endpoint must follow the same structure as the others. **i.e:** The whole REST endpoint must be specified like the previous example: `GET /services/connections/:id/apiInfo `.

Also, use the same place-holders used in other places _(like :id)_ instead of the keyword from the java classes.